{"name": "Test job companies from the company bunge website against Peviitor", "status": "failed", "statusDetails": {"message": "AssertionError: An unknown error occured in the API job company name test case", "trace": "get_job_details = ([], [], [], [], [])\nscrape_ui = (<tests.Test_Selenium_Suite.seleniumutils.TestUtils object at 0x7f276a1c0bb0>, (['Bunge Romania Opportunities', 'Junio...53a8350955f38e13e\", element=\"f.8B76384BE06D7C5BE3C55533FA5A51EE.d.61CC787BC689D008B8DA9C01120E6289.e.54\")>, ...], ...))\n\n    @pytest.mark.regression\n    @pytest.mark.ui\n    def test_bunge_company_ui(get_job_details, scrape_ui):\n        allure.dynamic.title(f\"Test job companies from the company {company_name} website against Peviitor\")\n    \n        scraped_jobs_data = get_job_details\n        with allure.step(\"Step 1: Get expected job company names\"):\n            job_companies_scraper = [company_name] * len(scraped_jobs_data[4])\n        with allure.step(\"Step 2: Get actual job companies from the Peviitor\"):\n            # Grab the data from peviitor ui\n    \n            testutils, job_details = scrape_ui[0], scrape_ui[1]\n            job_companies_peviitor = job_details[3]\n    \n        with allure.step(f\"Step 3: Compare job companies from the company {company_name} website against Peviitor\"):\n            allure.attach(f\"Expected Results: {job_companies_scraper}\", name=\"Scraper Expected Results\")\n            allure.attach(f\"Actual Results: {job_companies_peviitor}\", name=\"Peviitor Actual Results\")\n>           testutils.check_ui_job_company(job_companies_scraper, job_companies_peviitor)\n\ntests/Test_Selenium_Suite/test_bunge.py:152: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.Test_Selenium_Suite.seleniumutils.TestUtils object at 0x7f276a1c0bb0>\nexpected_company_name = []\nactual_company_name = ['bunge', 'bunge', 'bunge', 'bunge', 'bunge', 'bunge', ...]\n\n    def check_ui_job_company(self, expected_company_name: list, actual_company_name: list):\n    \n        # Lowercase all items in list to ensure proper comparison\n        expected_company_name, actual_company_name = [name.lower() for name in expected_company_name], [name.lower() for name in actual_company_name]\n    \n        # If no actual company name is in the API\n        if not actual_company_name:\n            msg = f\"No results display the company name within the API Response\"\n            allure.step(msg)\n            raise AssertionError(msg)\n    \n        # If the actual company name in the api does not correspond to expected company name raise error\n        for actual_name, expected_name in zip(actual_company_name, expected_company_name):\n            if actual_name != expected_name:\n                msg = f\"Company name does not match for one of the job results from the API Response\"\n                allure.step(msg)\n                raise AssertionError(msg)\n    \n>       assert expected_company_name[:len(actual_company_name)] == actual_company_name, \"An unknown error occured in the API job company name test case\"\nE       AssertionError: An unknown error occured in the API job company name test case\n\ntests/Test_Selenium_Suite/seleniumutils.py:293: AssertionError"}, "steps": [{"name": "Step 1: Get expected job company names", "status": "passed", "start": 1714064112623, "stop": 1714064112623}, {"name": "Step 2: Get actual job companies from the Peviitor", "status": "passed", "start": 1714064112623, "stop": 1714064112623}, {"name": "Step 3: Compare job companies from the company bunge website against Peviitor", "status": "failed", "statusDetails": {"message": "AssertionError: An unknown error occured in the API job company name test case\n", "trace": "  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_Selenium_Suite/test_bunge.py\", line 152, in test_bunge_company_ui\n    testutils.check_ui_job_company(job_companies_scraper, job_companies_peviitor)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_Selenium_Suite/seleniumutils.py\", line 293, in check_ui_job_company\n    assert expected_company_name[:len(actual_company_name)] == actual_company_name, \"An unknown error occured in the API job company name test case\"\n"}, "attachments": [{"name": "Scraper Expected Results", "source": "d7b58e2e-7ca5-47d4-b49b-ecdcb132add8-attachment.attach"}, {"name": "Peviitor Actual Results", "source": "49470db5-72b6-47ae-9eb2-3ec36074aa57-attachment.attach"}], "start": 1714064112623, "stop": 1714064112623}], "start": 1714064112623, "stop": 1714064112623, "uuid": "e160726a-4bd1-4e18-8d0e-64e8fdcd85bb", "historyId": "666d818c0f4dd3fa02b67ae48f526341", "testCaseId": "666d818c0f4dd3fa02b67ae48f526341", "fullName": "Test_Selenium_Suite.test_bunge#test_bunge_company_ui", "labels": [{"name": "tag", "value": "ui"}, {"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "Test_Selenium_Suite"}, {"name": "suite", "value": "test_bunge"}, {"name": "host", "value": "fv-az1430-770"}, {"name": "thread", "value": "2026-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_Selenium_Suite.test_bunge"}]}